cmd/jetstream-controller/main.go

```diff
@@ -129,6 +161,45 @@ func run() error {
 	return ctrl.Run()
 }
 
+func runControlLoop(config *rest.Config, natsCfg *controller.NatsConfig, controllerCfg *controller.Config) error {
+	// Setup scheme
+	scheme := runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	utilruntime.Must(v1beta2.AddToScheme(scheme))
+
+	mgr, err := ctrl.NewManager(config, ctrl.Options{
+		Scheme: scheme,
+		Logger: klog.NewKlogr().WithName("controller-runtime"),
+	})
+	if err != nil {
+		return fmt.Errorf("unable to start manager: %w", err)
+	}
+
+	if controllerCfg.CacheDir == "" {
```

Comment:
This assumes that if custom cache dir is passed by a user, it we already be there?
Maybe it's worth also trying to create it if it doesn't exist?